src/unk_020474B8.c

```diff
@@ -491,14 +491,14 @@ BOOL ScrCmd_BufferItemNamePlural(ScriptContext *ctx)
     return FALSE;
 }
 
-BOOL ScrCmd_33E(ScriptContext *param0)
+BOOL ScrCmd_BufferUndergroundGoodName(ScriptContext *param0)
```

Comment:
```suggestion
BOOL ScrCmd_BufferUndergroundGoodName(ScriptContext *ctx)
```
And in the header file.